src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -79,7 +79,7 @@ public void Initialize(EntityInfo entity, Func<IncomingMessageDetails, ReceiveCo
             options.ExceptionReceived += OptionsOnExceptionReceived;
 
             internalReceivers = new IMessageReceiver[numberOfClients];
-            dispatcher = new MultiProducerConcurrentDispatcher<Guid>(5000, TimeSpan.FromMilliseconds(500), 2, numberOfClients);
+            dispatcher = new MultiProducerConcurrentDispatcher<Guid>(1000, TimeSpan.FromMilliseconds(500), 6, numberOfClients);
```

Comment:
@Particular/azure-service-bus-maintainers would it hurt to complete only every second?